watsononlinestore/watson_online_store.py

```diff
@@ -282,32 +283,29 @@ def handle_message(self, message, channel):
             # self.discovery_client):
             return self.handle_DiscoveryQuery()
 
-        if ('send_no_input' in self.context.keys() and
-                self.context['send_no_input'] == 'yes'):
+        if ('get_input' in self.context.keys() and
+                self.context['get_input'] == 'no'):
             return False
 
         return True
 
     def add_test_users_to_DB(self):
-        molly = OnlineStoreCustomer(email="molly@gmail.com",
-                                    first_name="Molly",
-                                    last_name="DA",
-                                    purchase_history="abc123",
-                                    favorites="shoes",
-                                    logged_in=True)
-        self.cloudant_online_store.add_customer_obj(molly)
-        scott = OnlineStoreCustomer(email="scott@gmail.com",
+        rich = OnlineStoreCustomer(email="rich.hagarty@ibm.com",
+                                   first_name="Rich",
+                                   last_name="Hagarty",
+                                   shopping_cart="",
+                                   logged_in=True)
+        self.cloudant_online_store.add_customer_obj(rich)
+        scott = OnlineStoreCustomer(email="scott.dangelo@ibm.com",
```

Comment:
kind of a bad practice to put real IBM ID's out there.  Can we just start with no users and run once with the create path and then the 2nd run shows the recognition path?